src/index.js

```diff
@@ -21,6 +22,7 @@ class Orejs {
     Object.assign(this, eos)
     Object.assign(this, instrument)
     Object.assign(this, ore)
+    Object.assign(this, oreStandardToken)
```

Comment:
How does the oreStandardToken differ from token? Should all of these methods be inside of the token module?

Reply:
;)